### PR TITLE
Update talking_npc.py

### PR DESCRIPTION
**Pull request by grungies1138**
#### Other info (issues closed, discussion etc)

updating talking_npc to use the current Menu system

### CODE REVIEW

File: evennia/contrib/talking_npc.py [hunk 1]
```diff
@@ -1,122 +1,123 @@
...
+ 
+    return text, options
+ 
+ 
+def info1(caller):
+    text = "Oh, Evennia is where you are right now! Don't you feel the power?"
+ 
+    options = ({"desc": "Sure, *I* do, not sure how you do though. You are just an NPC.",
+                "goto": "info3"},
+               {"desc": "Sure I do. What's yer name, NPC?"},
```
Comment (Griatch) [inline evennia/contrib/talking_npc.py]: You seem to be missing a `goto` destination to "info2" here.

File: evennia/contrib/talking_npc.py [hunk 2]
```diff
@@ -1,122 +1,123 @@
 """
-
 Evennia Talkative NPC
-
 Contribution - Griatch 2011
```
Comment (Griatch) [inline evennia/contrib/talking_npc.py]: You can change this to "Contribution - Griatch 2011, grungies1138 2016" :)

**Comment by grungies1138** [comment]
fixed the missing goto issue.

**Comment by Griatch** [comment]
Thanks! Merged with a rebase.

**Pull request closed without merging**